testing/asserts.ts

```diff
@@ -1,6 +1,9 @@
 // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
 
-/** A library of assertion functions.
+/**
+ * @deprecated (will be removed after 0.204.0) Import from `std/asserts` instead.
```

Comment:
We discussed about the removal version in the core team, and we consider these APIs are too common to remove soon. We'd prefer to set the removal version to 1.0.0 (We expect 1.0.0 will happen this year)